api/agent/agent_options_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+package agent
+
+import (
```

Comment:
please format the imports as describe [here](https://github.com/eclipse-kanto/update-manager/pull/13#discussion_r1262090854)

Reply:
latest commits reverted the formatting as it was initially